lib/internal/errors.js

```diff
@@ -821,8 +821,14 @@ function hideInternalStackFrames(error) {
 // to make usage of the error in userland and readable-stream easier.
 // It is a regular error with `.code` and `.name`.
 class AbortError extends Error {
-  constructor() {
-    super('The operation was aborted');
+  constructor(message = 'The operation was aborted', options = {}) {
+    if (options === null || typeof options !== 'object') {
+      throw new codes.ERR_INVALID_ARG_TYPE('options', 'Object', options);
+    }
```

Comment:
The `validateObject()` validator can be used here instead for consistency.

Reply:
Not easily. validators.js imports errors.js and I'd rather avoid dancing around the circular reference.